Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:10
+WORKDIR /opt/app
+COPY package*.json ./
+RUN npm install
```

Comment:
Should this not be `npm install --production`?

Reply:
`ng` is required to build the project which is a dev dependency. We could use multi stage build. I envision this to be replaced by a web server like nginx. What do you think?

Reply:
I agree with moving to nginx.
@subbu3012 - wdyt?

Reply:
@vraravam @arunvelsriram i think i need some context here. what do you want to replace with ngnix?

Reply:
@subbu3012  - the thought process is whether to have `ng` as a runtime dependency for production-mode or whether we can create static assets that can be served via an nginx process.

Reply:
@vraravam In dev mode, angular uses `webpack-dev-server` internally which is wrapped under `ng serve` command. i dont think we need(and have) this in production. 

In production,`aws-amplify` is taking care of serving the static assets from `dist` directory. 

if you think we have more to discuss on this, we(you, @arunvelsriram, and i) can set up a small call and do so.

Reply:
In the call we decided that we will nginx for
1. serving frontend (static website)
2. acting as a proxy for backend